index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+  </head>
+
+  <body>
+
+    <nav>
+      <ul>
+        <li><a href="#welcome-section"> Welcome </a></li>
+        <li><a href="#works">Work</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
```

Comment:
I think you can explain also why you suggested that so instead of saying:

> I think it would be better if you wrap the `nav` tag in `header` tag.

You can say:
 
>  I think it would be better if you wrap the `nav` tag in the `header` tag. because using nav inside of a header is better as it's compliant with semantic HTML. Semantic HTML is necessary because they help a lot in screen readers and semantic HTML also helps in boosting page rank on google.